nodes/kitchen-node/src/service.rs

```diff
@@ -1,20 +1,13 @@
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
 
-use std::sync::Arc;
-use std::time::Duration;
-use sc_client::LongestChain;
-use sc_client_api::ExecutorProvider;
-use runtime::{self, opaque::Block, RuntimeApi};
-use sc_service::{error::{Error as ServiceError}, AbstractService, Configuration, ServiceBuilder};
+// use std::sync::Arc;
+// use sc_client::LongestChain;
+// use sc_client_api::ExecutorProvider;
```

Comment:
are these lines left here intentionally?

Reply:
The story behind these lines is that they would be necessary lower in the file in the `new_light` function. For reasons unknown to me, I can't get that block to compile anytime I use manual seal or instant seal. You can see my attempt commented out. I was hoping to get feedback from someone who is more familiar with consensus, but it might not come for a while. I consider this PR significant enough that I might disable the light clients, merge, and open a follow-up issue.

Then the question arises about whether I should keep the commented code, or remove it. It looks cleaner without commented code, but it will be less of a burden on whoever might give me help with the issue if I leave it.

Reply:
Okay, I've opened an issue to track the light client problem that caused me to comment these lines https://github.com/substrate-developer-hub/recipes/issues/241